butterknife-compiler/src/main/java/butterknife/compiler/BindingClass.java

```diff
@@ -393,10 +381,19 @@ private MethodSpec createBindMethod() {
       }
     }
 
+    // Finally return the unbinder
+    if (hasParentBinding() || hasViewBindings()) {
+      result.addStatement("return unbinder");
+    } else {
+      result.addStatement("return butterknife.ButterKnife.NOP_VIEW_UNBINDER");
```

Comment:
ButterKnife isn't on the classpath of the compiler module. Can do a `ClassName` representation of it though

6f7b05819d73b682397ba891c367a326ed882b0d